File: Iniciante/URI 1963.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val A: Double = leitor.nextDouble();
    val B: Double = leitor.nextDouble();
    
    val aumento: Double = (B * 100.0 / A) - 100.0;
    println(String.format("%.2f%%", aumento));
}

File: Iniciante/URI 1963.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val A = leitor.nextDouble()
        val B = leitor.nextDouble()
        val aumento = (B * 100.0 / A) - 100.0
        printf("%.2f%%\n", aumento)
    }
	
}